### ISSUE TITLE
Using breakpoints doesn't work when debugging

### ISSUE THREAD

**Issue by tyronedougherty** [kmagiera/babel-watch#60]
My npm script is as follows:

```
{
  ...
  "scripts": {
    "start": "babel-watch --debug 5859 src/backend/index.js",
    ...
  },
  ...
}
```
I get the `Debugger listening on [::]:5859` message and my debugger is able to connect on port `5859`, but putting breakpoints in my IDE (JetBrains WebStorm) appears to do nothing, making debugging effectively useless.

Using the --debug-brk option also screws up the debugging process, making it break on the underlying `runner.js` file instead of the script we want it to run on.

Any ideas how to make breakpoints work?

`babel-watch` version is `2.0.6`

**Comment by zaaack** [comment]
https://github.com/kmagiera/babel-watch/pull/62#issuecomment-313887432

I think this PR can fix.

**Comment by tyronedougherty** [comment]
@zaaack woohoo!! Can confirm this PR fixes it! Will comment in the PR too so hopefully it can get merged ASAP.

**Comment by iyobo** [comment]
Progress on this?

**Comment by marcioaffonso** [comment]
Same issue here. Could this PR be merged?

https://github.com/kmagiera/babel-watch/pull/62#issuecomment-313887432

**Comment by STRML** [comment]
#62 merged via #118 and released in `7.3.0`.
